src/redux/slices/evaluation-administration-slice.ts

```diff
@@ -140,9 +140,11 @@ export const publishEvaluationAdministration = createAsyncThunk(
 
 export const reopenEvaluationAdministration = createAsyncThunk(
   "evaluationAdministration/reopen",
-  async (id: number, thunkApi) => {
+  async ({ id, endDate }: { id: number | string; endDate: Date | string }, thunkApi) => {
```

Comment:
Put this as an interface (we can call this ReopenData) inside `evaluation-admnistration-type`

Reply:
You can check how it is done for `sendReminder`

Reply:
done :3